matlab/test/tfeather.m

```diff
@@ -282,3 +269,9 @@ function UnicodeVariableNames(testCase)
 
     end
 end
+
+function t = createFeatherCompatibleTable()
+    import arrow.internal.test.tabular.createTableWithSupportedTypes
+    t = createTableWithSupportedTypes;
+    t = removevars(t, "duration");
```

Comment:
Could you add a comment here saying that Feather V1 does not support the `Time[x][y]` types?